package.json

```diff
@@ -16,7 +16,8 @@
   "typings": "dist/datetimepicker.module.d.ts",
   "scripts": {
     "prepare": "npm run build",
-    "build": "rimraf dist && tsc"
+    "ngc": "./node_modules/.bin/ngc -p tsconfig.json",
+    "build": "rimraf dist && ngc"
```

Comment:
I am getting 

```
sh: 1: ngc: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! ng2-eonasdan-datetimepicker@0.0.9 build: `rimraf dist && ngc`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the ng2-eonasdan-datetimepicker@0.0.9 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/siatkowskim/.npm/_logs/2018-03-16T10_47_36_336Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ng2-eonasdan-datetimepicker@0.0.9 prepare: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ng2-eonasdan-datetimepicker@0.0.9 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

when I try to `npm install` now.

Reply:
I think you need to add `ngc` to `devDependencies` or something?

Reply:
I thought it's already installed, it seems not the case, I have it in my global so I've added @angular/compiler-cli in devDependencies, should be fine now.

Reply:
yeah works now